packages/codemods/src/plugins/default.ts

```diff
@@ -14,6 +14,34 @@ const defaultCodemodPlugin: CodemodPlugin = {
         return newImport;
       },
 
+      buildAttributes({ originalNode, transformedNode, composedNode }) {
```

Comment:
other ones use `currentNode`, should we stick to similar naming maybe?

Reply:
also shouldn't the signature match `BuildAttributesContext`?

Reply:
👍